openlibrary/plugins/worksearch/languages.py

```diff
@@ -40,7 +41,8 @@ def process_key(self, key):
         return key.replace("_", " ")
 
 
-def get_top_languages(limit):
+@cache.memoize(engine="memcache", expires=5 * 60)
+def get_top_languages(limit: int):
```

Comment:
What is the return value?
```suggestion
def get_top_languages(limit: int) -> list[str]:
```